proposals/checked-user-defined-operators.md

```diff
@@ -4,9 +4,9 @@
 [summary]: #summary
 
 C# should support defining `checked` variants of the following user-defined operators so that users can opt into or out of overflow behavior as appropriate:
-*  The `++` and `--` unary operators (https://github.com/dotnet/csharplang/blob/main/spec/expressions.md#postfix-increment-and-decrement-operators and https://github.com/dotnet/csharplang/blob/main/spec/expressions.md#prefix-increment-and-decrement-operators).
-*  The `-` unary operator (https://github.com/dotnet/csharplang/blob/main/spec/expressions.md#unary-minus-operator).
-*  The `+`, `-`, `*`, and `/` binary operators (https://github.com/dotnet/csharplang/blob/main/spec/expressions.md#arithmetic-operators).
+*  The `++` and `--` unary operators [§11.7.14](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/expressions.md#11714-postfix-increment-and-decrement-operators) and [§11.8.6](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/expressions.md#1186-prefix-increment-and-decrement-operators).
```

Comment:
Every time I make reference to the spec now I'm going to have to go and fine one of these `§` symbols and copy/paste it over now :stuck_out_tongue:

Reply:
pro tip: copy from the [detailed table of contents](https://github.com/dotnet/csharpstandard/tree/draft-v6/standard)